.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release Workflow
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1.0.0
```

Comment:
It is not maintained anymore: https://github.com/actions/create-release

Reply:
What about this PR @mshudrak

Reply:
I don't think Jenkins is a good option here.

Reply:
> I don't think Jenkins is a good option here.

I think then we can use GitHub REST API. what do you think?

Reply:
I think then we can use GitHub REST API. what do you think @mshudrak , is it ok?

Reply:
> I think then we can use GitHub REST API. what do you think @mshudrak , is it ok?
for using the GitHub REST API, I need the Asset file path, can you help me with that? as I can't able to locate that?

Reply:
> 

@mshudrak can I use GitHub REST API for,  I need the Asset file path, can you help me with that? as I can't able to locate that?

Reply:
I think we have #100 and #98 already for this feature. We will not consider any more PRs on this issue. Sorry.